### ISSUE TITLE
Tracking issue for GitHub/Gitea mirroring

### ISSUE THREAD

**Issue by dboreham** [cerc-io/hosting#32]
Repositories can be mirrored between GitHub and Gitea. Mirroring essentially denotes one "side" as read-only. Writes (code commits, issues, PRs, CI, releases) are done on the writable side. 

Note that at present only code is actually synced between mirrored repos (even though the UI suggests that issues and so on will sync, they actually do not).

A mirrored repo can be set up as writable on GitHub or writable on Gitea.
In the case of a mirrored repo that is writable at GitHub, Gitea knows this and disables writes on the Gitea side (kind of-- you can still create issues for example on the Gitea site) but GitHub has no clue that the repo is mirrored. So you'd have to set up access control at Github to disable writes, and add readme text to let users know that this is a mirror repo with a link to the writable Gitea repo.

In short mirroring is useful to maintain a usable place at GitHub for users to get the project's code and to host doc.
It's also useful to allow sovereign hosted build environment to run under Gitea that has been seeded from Github.

**Comment by dboreham** [comment]
With a writable Gitea repo mirrored to GitHub, GitHub users can fork and create PRs.
We need to determine some workflow for taking such a PR and transforming it into a PR against the Gitea repo, or alternatively decide that it will be sufficient to tell contributors to re-cut their PRs against Gitea directly.

**Comment by dboreham** [comment]
Closing since we understand GH->GT migration at this point.
